nautobot_device_lifecycle_mgmt/api/serializers.py

```diff
@@ -152,7 +157,7 @@ class Meta:
 
 
 class ValidatedSoftwareLCMSerializer(
-    CustomFieldModelSerializer, TaggedObjectSerializer
+    TaggedObjectSerializer, CustomFieldModelSerializer
```

Comment:
No, this change is intentional. Tags won't work when using the API serializers unless the `TaggedObjectSerializer` is inherited prior to the `CustomFieldModelSerializer`. You can verify this by trying to POST/PUT/PATCH one of the objects with a tag with its current configuration.

If you'd like, I can separate this out into its own issue and PR, but since I discovered it when I copy/pasting previous serializers I just bundled that fix into this PR.

Reply:
Ah, gotcha, makes sense.